src/python/pants/backend/javascript/subsystems/nodejs.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Iterable
+
+from pants.core.util_rules.external_tool import (
+    DownloadedExternalTool,
+    ExternalToolRequest,
+    TemplatedExternalTool,
+)
+from pants.engine.fs import EMPTY_DIGEST, Digest, MergeDigests
+from pants.engine.platform import Platform
+from pants.engine.process import Process
+from pants.engine.rules import Get, Rule, SubsystemRule, collect_rules, rule
+from pants.engine.unions import UnionRule
+from pants.util.logging import LogLevel
+
+
+class NodeJS(TemplatedExternalTool):
+    options_scope = "nodejs"
+    help = "The NodeJS Javascript runtime (including npm and npx)."
+
+    # TODO: Update to latest LTS
```

Comment:
Do we use latest/greatest (18.1.0) - or latest stable (16.15.0 LTS)?

Reply:
Probably LTS, given that users can change this. 

Thoughts on how this subsystem will work long term? Does it determine the JS used for users own code, or only tools users run like Prettier?

Reply:
On the fence. I'm prepping a Google doc or Github discussion to talk more about compilation options, once I get some `typescript_sources` in mainline. That'll cover this subsystem as well. 

There are pros/cons - and I have them segmented by user-base/use case, but it needs more eyeballs than just one person's opinion.